tests/app/views/contexts/summary/test_question.py

```diff
@@ -969,3 +1047,58 @@ def test_dynamic_answer_options(self):
 
                 # Then
                 self.assertEqual(question.answers[0]["value"], expected_output)
+
+    def test_get_answer_method(self):
```

Comment:
We should have one to ensure the answer is also escaped.

Reply:
Added one test for testing escaped result, thanks!